README.md

```diff
@@ -1,2 +1,10 @@
 # fabrictestbed-extensions
+
 Extensions for the FABRIC API/CLI.  
+
+## Install Instructions
+```
+git clone https://github.com/fabric-testbed/fabrictestbed-extensions.git 
```

Comment:
You probably should start pushing to PyPi at this point. PyPi respects the 3-level versioning convention with 'b' or 'rc' release candidate options, so you can call this 1.0.0b1 or 1.0.0rc1